truffle-config.js

```diff
@@ -1,5 +1,38 @@
+const path = require('path');
+
+const { NearProvider } = require('near-web3-provider');
+
+function NearLocalProvider() {
```

Comment:
actually, should we also put these two functions directly into near web3 provider?
It seems like everyone will need them.
Also `networkId` as local still seems more sense to me. That said our networkId has a really weird meaning (it was suppose to be matching chainID that we actually have from the node, but it doesn't - because local networks are actually have test-<random> chainIDs).

Reply:
moved this into near-web3-provider and changed the name back to `local`. I kind of like the idea of using a unique networkId, maybe we'll use it on another project. It's a great way to have a separate directory that in no way will mess with an environment. You can reasonably delete the `balancer-tests` directory without ever affecting any `local` private keys. But I'm not stuck on this, I just think it's neat. Marking as resolved, and thanks for the suggestion on this. Makes so much sense, but I didn't think to move it over there.